src/org/usfirst/frc4904/robot/commands/ShootCycle.java

```diff
@@ -0,0 +1,15 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class ShootCycle extends CommandGroup {
+	public ShootCycle() {
+		super("ShootCycle");
+		requires(RobotMap.Component.shooter);
+		addParallel(new Align());
+		addSequential(new FlywheelSpinup());
+		addSequential(new Shoot());
```

Comment:
While it does initialize the shoot command before the it is aligned, the execution of the command is not included, so I don't think we have any reason to believe it would shoot before it aligns.